Tests/Build/Utilities.swift

```diff
@@ -0,0 +1 @@
+../Functional/Utilities.swift
```

Comment:
yes it does. 
https://github.com/apple/swift-package-manager/blob/master/Tests/ManifestParser/Utilities.swift